app/templates/administrative-units/index.hbs

```diff
@@ -140,20 +140,7 @@
         {{#if this.hasErrored}}
           <TableMessage::Error />
         {{else if this.hasNoResults}}
-          <TableMessage>
-            <p>
-              Er werden geen zoekresultaten gevonden. Kijk na op spelfouten, of
-              probeer een andere zoekopdracht. Indien een organisatie ontbreekt,
-              meldt dit aan
-              <a
-                class="au-c-link"
-                href="mailto:{{config "contactEmail"}}"
-              >{{config "contactEmail"}}</a>
-              of
-              <AuLink @route="administrative-units.new">maak een organisatie
-                aan.</AuLink>
-            </p>
-          </TableMessage>
+          <TableMessage::NoResult />
```

Comment:
I think here you should revert it to the previous version, when we are in the administrative unit page it makes no sense to provide the option to create a new person and I don't see it in the designs